test/feature/Yield/ReturnGenerator.js

```diff
@@ -0,0 +1,92 @@
+function* f1() {
+  yield 1;
+  yield 2;
+}
+
+var g1 = f1();
+assert.deepEqual(g1.next(), {value: 1, done: false});
+assert.deepEqual(g1.return(42), {value: 42, done: true});
+assert.deepEqual(g1.next(), {value: undefined, done: true});
+
+function* f2() {
+  yield 1;
+  try {
+    yield 1;
```

Comment:
I think @arv just wants unique values for paths out of the test to aid in debugging.  Sometimes an error message is confusing but it has a piece of unique data that leads us to finding the problem.